tests/acceptance/item-inline-edit-test.js

```diff
@@ -0,0 +1,216 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import '../factories/orders_package';
+import '../factories/designation';
+import '../factories/item';
+import '../factories/location';
+import FactoryGuy from 'ember-data-factory-guy';
+import TestHelper from 'ember-data-factory-guy/factory-guy-test-helper';
+import { mockFindAll } from 'ember-data-factory-guy';
+
+var App, pkg;
+
+module('Acceptance: Item inline edit', {
+  beforeEach: function() {
+    App = startApp({}, 2);
+    TestHelper.setup();
+    var location = FactoryGuy.make("location");
+    pkg = FactoryGuy.make("item", { id: 50, state: "submitted" , quantity: 10, height: 10, width: 15, length: 20, notes: "Inline edit test" });
+    mockFindAll('location').returns({json: {locations: [location.toJSON({includeId: true})]}});
```

Comment:
`FactoryGuy.make("item", { id: 50, state: "submitted" , quantity: 10, height: 10, width: 15, length: 20, notes: "Inline edit test" })` this line can be avoided by adding it to factory and if parameters like height, width and length are not used for testing they can be instead set to default values. Also it would be better if you use line break for such a long lines.